packages/@okta/vuepress-site/docs/guides/refresh-tokens/get-refresh-token/index.md

```diff
@@ -14,15 +14,15 @@ The only flows that support refresh tokens are the resource owner password flow
 
 > **Note:** The authorization code flow is unique in that the `offline_access` scope must be requested as part of the code request to the `/authorize` endpoint and not the request sent to the `/token` endpoint.
 
-### Get a refresh token with the code flow
+> **Note:** Whether persistent refresh token or rotating refresh token behavior is enabled depends on what type of application that you are using. When you select **Refresh Token** as an allowed grant type, [SPAs use refresh token rotation](#renew-access-and-id-tokens-with-spas) as the default behavior. Native apps and web apps use persistent refresh token behavior as the default. See [Refresh token rotation](/docs/guides/refresh-tokens/refresh-token-rotation).
 
-In the case of the authorization code flow, you use the authorization server's `/authorize` endpoint to get an authorization code, specifying an `offline_access` scope. You then send this code to the `/token` endpoint to get an access token and a refresh token.
+## Get a refresh token with the code flow
 
-> **Note:**  Authorization code with PKCE requests don't return refresh tokens if they are sent from SPAs or other browser-based apps. Instead, you can [silently refresh tokens](#get-a-new-access-token-id-token-silently-for-your-spa) by making a call to the `/authorize` endpoint.
+In the case of the authorization code flow, you use the Authorization Server's `/authorize` endpoint to get an authorization code, specifying an `offline_access` scope. You then send this code to the `/token` endpoint to get an access token and a refresh token.
 
 See [Obtain an authorization grant from a User](/docs/reference/api/oidc/#authorize) and [Implementing the authorization code flow](/docs/guides/implement-auth-code/) for more information on the `/authorize` endpoint and the authorization code flow.
 
-#### Example request for an authorization code and refresh token
+### Example request for an authorization code and refresh token
 
```

Comment:
Should we add another example for `authorization code with PKCE and refresh token` and similarly for token endpoint as well ? SPAs will be using this flow

Reply:
Or we should link to the PKCE section